code-challenges.test.js

```diff
@@ -9,37 +9,95 @@
 
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
+//_______________________________________________________________________________________________________________
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
+// // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
+describe("multBy3", () => {
+    it("takes in an array of numbers and returns an array with all the numbers multiplied by 3.", () => {
+      const numbersArray1 = []
+      const numbersArray2 = []
+      expect(multBy3(numbersArray1)).toEqual([])
+      expect(multBy3(numbersArray2)).toEqual([])
+    })
+  })
+
+
+// Pseudo Code: 
+// Create a function called multBy3
+// input a param of array 
+// return the array and use the method map() to access the indexes
+// multiply the values within the array by 3
+
 
 // b) Create the function that makes the test pass.
+const multBy3 = (array) => {
+  return array.map(value => value * 3)
+}
 
+// OUTPUT:
+// ✓ takes in an array of numbers and returns an array with all the numbers multiplied by 3. (1 ms)
 
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+// Snapshots:   0 total
 
+// // //_________________________________________________________________________________________________________________
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
-
 // a) Create a test with expect statements for each of the variables provided.
 
 const num1 = 15
-// Expected output: "15 is divisible by three"
+// // Expected output: "15 is divisible by three"
 const num2 = 0
-// Expected output: "0 is divisible by three"
+// // Expected output: "0 is divisible by three"
 const num3 = -7
-// Expected output: "-7 is not divisible by three"
-
+// // Expected output: "-7 is not divisible by three"
+
+describe("divisibleBy3", () => {
+    it("takes in a number as an argument and decides if the number is evenly divisible by 3 or not", () => {
+      const num1 = []
+      const num2 = []
+      const num3 = []
+      expect(divisibleBy3(num1))
+      expect(divisibleBy3(num2))
+      expect(divisibleBy3(num3))
+    })
+  })
```

Comment:
The setup for this test looks great, however, this test is not comparing against and expected output which will result in a guaranteed pass.  Make sure to include a .toEqual with your expected output such as: `expect(divisibleBy3(num1)).toEqual("divisble by 3")`